patches/react-native-notifications+2.1.7.patch

```diff
@@ -1,15 +1,27 @@
 diff --git a/node_modules/react-native-notifications/android/app/src/main/AndroidManifest.xml b/node_modules/react-native-notifications/android/app/src/main/AndroidManifest.xml
-index 7053040..2a97b04 100644
+index 7053040..ad63eff 100644
 --- a/node_modules/react-native-notifications/android/app/src/main/AndroidManifest.xml
 +++ b/node_modules/react-native-notifications/android/app/src/main/AndroidManifest.xml
-@@ -29,6 +29,7 @@
+@@ -11,6 +11,7 @@
+         android:protectionLevel="signature" />
+     <uses-permission android:name="${applicationId}.permission.C2D_MESSAGE" />
+     <uses-permission android:name="android.permission.VIBRATE" />
++    <uses-permission android:name="android.permission.WAKE_LOCK" />
```

Comment:
I was trying to find where this permission gets used, because it can drain battery in activities. I don't believe it's being used in any on-screen Activity, but I couldn't find where this permission became needed.

Reply:
this is needed for the Push notification scheduler, there is a receiver that uses AlarmManager to set up the schedule.